libraries/react-experimental/src/advanced-tests.js

```diff
@@ -36,31 +36,62 @@ app.id = "app";
 document.body.appendChild(app);
 let scratch; // This will hold the actual element under test.
 
+let reactRoot = null;
+function render(element) {
+  act(() => {
+    reactRoot.render(element);
+  });
+}
+
+before(() => {
+  window.IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
Should this be unset in after()?

Reply:
Yep: [c50131cddda59935bae5f8c38b65ecde151ab7b4](https://github.com/webcomponents/custom-elements-everywhere/pull/1608/commits/c50131cddda59935bae5f8c38b65ecde151ab7b4)